ramls/eholdings.raml

```diff
@@ -552,20 +586,93 @@ types:
                       type: string
                       example: "https://hello.io"
                       required: false
+                    name:
+                      description: |
+                        Title name for a custom resource.
+                        Note that this attribute can be updated ONLY FOR A CUSTOM RESOURCE.
+                      type: string
+                      example: "Sample Title"
+                      required: false
+                    isPeerReviewed:
+                      description: |
+                        Peer review indicator for a custom resource.
+                        Note that this attribute can be updated ONLY FOR A CUSTOM RESOURCE.
+                      type: boolean
+                      example: true
+                      required: false
+                    publicationType:
+                      description: |
+                        Publication Type for a custom resource.
+                        Note that this attribute can be updated ONLY FOR A CUSTOM RESOURCE.
+                      type: string
+                      enum: ["All", "Audiobook", "Book", "Book Series", "Database", "Journal", "Newsletter", "Newspaper", "Proceedings", "Report", "Streaming Audio", "Streaming Video", "Thesis & Dissertation", "Website", "Unspecified"]
```

Comment:
I am not sure if "All" is a permitted Publication Type for custom resources. Verified that it is displayed in UI dropdown when we try to edit a custom resource, however, on selecting "All", we always get back a 400 saying "PubType" is required. Could you please verify whether "All" is permitted by RM API?

Reply:
👍, we might want to create a bug to change UI options in drop-down accordingly both when creating a custom resource and well as updating it. I can create that in JIRA.